src/OrleansAzureUtils/Hosting/AzureSilo.cs

```diff
@@ -99,12 +99,27 @@ public bool Start()
         /// Initialize this Orleans silo for execution with the specified Azure deploymentId and role instance
         /// </summary>
         /// <param name="config">If null, Config data will be read from silo config file as normal, otherwise use the specified config data.</param>
-        /// <returns><c>true</c> is the silo startup was successfull</returns>
+        /// <returns><c>true</c> is the silo startup was successful</returns>
         public bool Start(ClusterConfiguration config)
         {
+            return Start(null, config);
+        }
+
+        /// <summary>
+        /// Initialize this Orleans silo for execution with the specified Azure deploymentId and role instance
+        /// </summary>
+        /// <param name="deploymentId">Azure DeploymentId this silo is running under</param>
+        /// <param name="config">If null, Config data will be read from silo config file as normal, otherwise use the specified config data.</param>
+        /// <returns><c>true</c> is the silo startup was successful</returns>
+        public bool Start(string deploymentId, ClusterConfiguration config)
```

Comment:
Yes, but this preserves the order of the arguments as it used to be, rather than flipping them around. Making an optional parameter would mean it has to go at the end. I was also following the code style with the other overload (re line 103)